tests/unit/tasks/install-blueprint-test.js

```diff
@@ -1,9 +1,18 @@
 'use strict';
 
-const InstallBlueprintTask = require('../../../lib/tasks/install-blueprint');
+const RSVP = require('rsvp');
+const path = require('path');
+const fs = require('fs-extra');
 const td = require('testdouble');
 const SilentError = require('silent-error');
+
 const expect = require('../../chai').expect;
+const mkTmpDirIn = require('../../../lib/utilities/mk-tmp-dir-in');
+const InstallBlueprintTask = require('../../../lib/tasks/install-blueprint');
+
+let root = process.cwd();
```

Comment:
@stefanpenner What is wrong with process.cwd()?

Reply:
`__dirname` will always return a path to the file whereas `process.cwd` will return a path where `node` is executed. By using `__dirname` you will make sure to have a stable path at all times (as you start with a file and going up). If you use use `process.cwd`, you will get different paths.

```sh
mocha tests/unit/tasks/install-blueprint-test.js
# process.cwd() => /full/path/to/ember/cli
# __dirname => /full/path/to/ember/cli/tests/unit/tasks/
```

vs.

```sh
cd tests/unit/tasks/ && mocha install-blueprint-test.js
# process.cwd() => /full/path/to/ember/cli/tests/unit/tasks/
# __dirname => /full/path/to/ember/cli/tests/unit/tasks/
```

`tmp` directory will be created in different places and we want to avoid that. @stefanpenner correct me if I'm wrong.